types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// TypeScript Version: 3.4
+
+import {Plugin} from 'unified'
+
+interface RemarkFootnotesOptions {
+  inlineNotes?: boolean
```

Comment:
It can be helpful to include the description of the property as a TSDoc comment.
https://github.com/remarkjs/remark-footnotes#optionsinlinenotes

```suggestion
  /**
   * Whether to support `^[inline notes]`
   *
   * @defaultValue false
   */
  inlineNotes?: boolean
```